Realm.xcodeproj/project.pbxproj

```diff
@@ -2934,6 +2935,7 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				DEFINES_MODULE = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
We don't want to bump the deployment target for the framework, and all of these settings added to the xcodeproj were deliberately not there (they're set in the xcconfig files so that they can be shared between targets).